static/app/views/alerts/rules/details/body.tsx

```diff
@@ -110,186 +97,6 @@ export default class DetailsBody extends React.Component<Props> {
     });
   };
 
-  renderTrigger(label: string, threshold: number, actions: Action[]): React.ReactNode {
-    const {rule} = this.props;
-
-    if (!rule) {
-      return null;
-    }
-
-    const status =
-      label === AlertRuleTriggerType.CRITICAL
-        ? t('Critical')
-        : label === AlertRuleTriggerType.WARNING
-        ? t('Warning')
-        : t('Resolved');
-    const statusIcon =
-      label === AlertRuleTriggerType.CRITICAL ? (
-        <StyledIconDiamond color="red300" size="md" />
-      ) : label === AlertRuleTriggerType.WARNING ? (
-        <StyledIconDiamond color="yellow300" size="md" />
-      ) : (
-        <StyledIconDiamond color="green300" size="md" />
-      );
-
-    const thresholdTypeText = (
-      label === 'resolved'
-        ? rule.thresholdType === AlertRuleThresholdType.BELOW
-        : rule.thresholdType === AlertRuleThresholdType.ABOVE
```

Comment:
can we pull out this level of ternary into its on variable, kinda hard to read so many levels

Reply:
👍